doc/python/simulation.rst

```diff
@@ -274,29 +274,37 @@ Definitions
         current, mechanism state, and a number of other quantities, measured either over the whole cell,
         or at specific sites (see :ref:`pycablecell-probesample`).
 
-        Probes are described by probe addresses, and the collection of probe addresses for a given cell is
+    probeset address
+        Probes are described by probeset addresses, and the collection of probeset addresses for a given cell is
```

Comment:
I wouldn't say that probes are described by probeset addresses. Maybe "probesets are described by probeset addresses" but I really still don't see the distinction between the two, what's a probeset address? The `cell_member`? That is usually already called the _probeset id_.